config/routes.rb

```diff
@@ -3,7 +3,7 @@
   mount Blacklight::Oembed::Engine, at: 'oembed'
   mount Riiif::Engine => '/images', as: 'riiif'
   root to: 'spotlight/exhibits#index'
-  mount Spotlight::Engine, at: 'spotlight'
+  mount Spotlight::Engine, at: '/'
```

Comment:
Thanks...I wonder if the [order matters](https://github.com/sul-dlss/exhibits/commit/4d0f6eeb56ce3e636571cf66e8f3eee952a1d8e4#diff-21497849d8f00507c9c8dcaf6288b136) but glad to hear it's running smoothly so far.